internal/resources/resource_host.go

```diff
@@ -10,8 +10,8 @@ import (
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 
-	rest "github.com/hpe-hcss/quake-client/v1/pkg/client"
 	"github.com/HewlettPackard/hpegl-metal-terraform-resources/pkg/client"
+	rest "github.com/hpe-hcss/quake-client/v1/pkg/client"
```

Comment:
I thought this was public repo.  Should we eliminate hpe-hcss references?

Reply:
Still haven't moved the rest-client portion to this new Org. Will remove this after that. Just keeping it, for now, to be able to build and test the ongoing changes.